pkg/machines/components/vm/dummyVm.jsx

```diff
@@ -59,4 +59,7 @@ DummyVm.propTypes = {
     vm: PropTypes.object.isRequired,
 };
 
-export default DummyVm;
+export function asDummVms(vms, uiVms) {
+    const returnVal = uiVms.filter(uiVm => vms.find(vm => vm.name == uiVm.name && vm.connectionName == uiVm.connectionName) == undefined);
+    return returnVal;
```

Comment:
Code style: you can just return that value right away, it makes the first line even shorter.

Also, "asDummVms" is a weird function name, it looks like a typo. What's the "as"?